Description: Do not start lvmetad in containers
Author: Balint Reczey <rbalint@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/lvm2/+bug/1576341
Forwarded: no
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

Index: lvm2-2.03.07-1ubuntu1/scripts/lvm2_monitoring_systemd_red_hat.service.in
===================================================================
--- lvm2-2.03.07-1ubuntu1.orig/scripts/lvm2_monitoring_systemd_red_hat.service.in
+++ lvm2-2.03.07-1ubuntu1/scripts/lvm2_monitoring_systemd_red_hat.service.in
@@ -6,6 +6,7 @@
 Before=local-fs-pre.target shutdown.target
 DefaultDependencies=no
 Conflicts=shutdown.target
+ConditionVirtualization=!container
 
 [Service]
 Type=oneshot
